assets/app/components/site/pageListItem.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+const propTypes = {
+  page: React.PropTypes.object
+};
+
+class PageListItem extends React.Component {
+  getButtonCopy(page) {
+    return page.type === 'dir' ? 'Open' : 'Edit';
+  }
+
+  render() {
+    const { page } = this.props;
+
+    return (
+      <li className="list-group-item">
+        <div className="usa-grid">
+          <div className="usa-width-two-thirds">
+            {page.name}
+          </div>
+          <div className="usa-width-one-third">
+            <a className="usa-button usa-button-outline file-list-item-button" href={page.url}>
```

Comment:
We'll have the `:siteId` but not the `:branch`. The latter will have to be manually passed down. We'll only have the branch slug once the user navigates